charts/argo-cd/values.yaml

```diff
@@ -1725,6 +1725,9 @@ redisSecretInit:
   # @default -- `""` (defaults to global.priorityClassName)
   priorityClassName: ""
 
+  # -- Assign custom [affinity] rules to the Redis secret-init Job
+  affinity: |
```

Comment:
Everywhere else in the values file, affinity is defined as 

```
affinity: {}
```

So this should probably do the same for consistency.